src/main/java/com/realestate/service/user/service/UserService.java

```diff
@@ -87,7 +89,7 @@ public int generateSecretCode(UserEmailDto userEmailDto) {
 
     /** 6자리 난수 발생 */
     Random random = new Random();
-    int secretCode = random.nextInt(999999);
+    int secretCode = random.nextInt(1000000);
```

Comment:
보통 이러한 숫자를 매직 넘버라고 하는데 저번에도 말씀드렸지만 잘못 이해하신거 같아서요
이런 매직넘버는 어떤 의미인지 이해하기 어렵기 때문에 의미있는 상수로 선언해서 
사용하시는게 좋을것 같아요!

`private static final int 의미있는_이름_상수 = 10000;`

```suggestion
    int secretCode = random.nextInt(의미있는_이름_상수);
    
```